src/sd-boot.h

```diff
@@ -40,5 +41,7 @@ typedef struct sdb_entry_list {
 } sdb_entry_list_t;
 
 extern bool			sdb_get_entry_list(sdb_entry_list_t *result);
+extern bool			sdb_is_kernel(const char *application);
+extern const char *		sdb_get_next_kernel();
```

Comment:
Again, I'd use (void) in this declaration.

Reply:
Done